CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 Features:
 - Added support for the CMake Debugger. [#3093](https://github.com/microsoft/vscode-cmake-tools/issues/3093)
 
+## 1.14.35
```

Comment:
I think you can merge this with the 1.15 section. I don't think we're planning to do another patch release before 1.15.

Reply:
sounds good